pkgs/development/libraries/gdal/default.nix

```diff
@@ -2,7 +2,7 @@
 , postgresql, libmysqlclient, libgeotiff, pythonPackages, proj, geos, openssl
 , libpng, sqlite, libspatialite, poppler, hdf4, qhull, giflib, expat
 , libiconv, libxml2, autoreconfHook
-, netcdfSupport ? true, netcdf, hdf5, curl
+, netcdfSupport ? true, netcdf, hdf5, curl, pkg-config
```

Comment:
you've added `pkg-config`, but it is not used anywhere

Reply:
I had it inside nativeBuildInputs when I was testing. Seems like I have accidentally removed it before doing the commit. I just tested that without it won't compile. Let me test again and I will correct the commit.

Reply:
I have corrected this mistake.